doc/release-notes/release-5.sgml

```diff
@@ -151,6 +152,23 @@ Most user-facing changes are reflected in squid.conf (see below).
    yet do TLS-in-TLS.
 
 
+<sect1>OpenSSL 3.0 Support
```

Comment:
> https://github.com/squid-cache/squid/pull/1131/commits/038ae6bc6d19b49227103d5ff890f043f5202deb Consistent OpenSSL version labeling

The above commit missed at least three cases of a different version spelling in publicly visible Squid text AFAICT:

```
upstream/master:src/cf.data.pre:        Not supported in builds with OpenSSL v3 or newer.
upstream/master:src/ssl/support.cc:               "It is removed in Squids built with OpenSSL v3+.");
upstream/master:src/ssl/support.cc:        throw TextException("Cannot use ssl_engine in Squid built with OpenSSL v3+", Here());
```

Reply:
Done in [d8ec67a](https://github.com/squid-cache/squid/pull/1131/commits/d8ec67affdf7e58394ae001ba12a121aa80c574d)